### ISSUE TITLE
Custom Auto-Localization for Materials/Parts

### ISSUE THREAD

**Issue by Gliese-832-c** [ISurrealI/ContentCreator#15]
You should be able to enter a string somewhere to automatically localize parts. This should allow for:
- The localization of custom parts (For example, if you have a girder part, you should be able to tell the mod to localize it as "_some material_ Girder")
- The overriding of material localization (if you have a material that is called lead_lowgrade, that localizes into "Lead Lowgrade _some part_", wheras I want it to be localized to "Low-Grade Lead _some part_")
- The overriding of existing localization names for default parts like ingot, plate, etc.

**Comment by ISurrealI** [comment]
I think i will add a feature to add and change oredicts too. Because there're sometimes stupid stuff like aluminium/aluminum

**Comment by Gliese-832-c** [comment]
Agreed, that sounds useful. More options for customization are always good.

**Comment by ISurrealI** [comment]
did this too.
